api/authorization-api-1_0.md

```diff
@@ -427,7 +427,7 @@ The following is a non-normative example of an Access Evaluation Request:
 ~~~ http
 POST /access/v1/evaluation HTTP/1.1
 Host: pdp.mycompany.com
-Authorization: <myoauthtoken>
+Authorization: Bearer <myoauthtoken>
```

Comment:
note: we are not precluding other Authorization schems (like `basic <token>`)

with that said, since this example specifically states <myoauthtoken>, the change makes sense.